DEVELOPER_GUIDE.md

```diff
@@ -64,13 +64,13 @@ If it's the only version of node installed, it will automatically be set to the
 
 #### Install `yarn`
 
-To install yarn run:
+OpenSearch Dashboards is set up using yarn v1. To install it, run:
 
 ```bash
-$ npm i -g yarn
```

Comment:
Do we need to add back `$ npm i -g corepack`? Or is that no longer needed with our node updates?

Reply:
According to the node website,

> Added in: v16.9.0, v14.19.0

Reply:
As long as user is following the guide, they won't need to install it